src/compressor/QatAccel.h

```diff
@@ -15,15 +15,23 @@
 #ifndef CEPH_QATACCEL_H
 #define CEPH_QATACCEL_H
 
-#include <qatzip.h>
+#include <memory>
 #include <boost/optional.hpp>
 #include "include/buffer.h"
 
+struct QzSession_S; // typedef struct QzSession_S QzSession_T;
+
+struct QzSessionDeleter {
+  void operator() (QzSession_S *session);
+};
+
+using unique_session_ptr = std::unique_ptr<struct QzSession_S, QzSessionDeleter>;
```

Comment:
this alias is not a part of the public interface. could be moved into `QatAccel`.

Reply:
Sure and thanks for reminding